site/templates/common/sponsor-image.jet

```diff
@@ -0,0 +1,10 @@
+{{block sponsor(text, link, external, img)}}
+  <div class="sponsor">
+    {{if text}}
+      <span>{{text}}</span>
+    {{end}}
+    {{if link}}<a href="{{link}}" {{if external}} target="_blank" {{end}}>{{end}}
```

Comment:
Might be able to make this a bit neater by [wrapping while invoking](https://github.com/CloudyKit/jet/wiki/3.-Jet-template-syntax#wrapping-while-invoking).

Reply:
Hmm I tried it, It looks unnecessarily less readable with logic moved into an extra block and yield, also made the vars harder to follow